modules/ROOT/pages/design-publish.adoc

```diff
@@ -0,0 +1,47 @@
+= Publish an API Specification
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+When you want to share your API specification with other developers, you can publish it to Anypoint Exchange. Exchange creates a page that presents the project for anyone in your business group to view and download the project from.
+
+.The page that Exchange creates for a published project includes a left pane for viewing the specification's documentation, a middle pane for viewing a description of the specification, and a right pane for viewing metadata about the specification.
+image:apid-example-page-in-exchange.png[An example page for a specification published in Exchange]
+
+<1> The left pane, which you can navigate to view the documentation. The navigation is generated from the documentation in the specification.
+<2> The middle pane, which describes the specification. You write the description in Exchange after publishing the specification.
+<3> The right pane, which displays metadata about the specification.
+
+== Procedure
+
+. Open the API-specification project that you want to publish, if it is not already open.
+. In the top-right corner of API Designer, click *Publish* and then click *Publish to Exchange*.
+
+image:apid-publish-dialog-api-specs.png[]
+
+<1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project.
+<2> *API version*: The version number of the API specification. For a description of this field, see the next section.
```

Comment:
Please name the section rather than referring to it as "the next section," to help keep topics independent organization. Just a good habit to prepare for potential reuse or reorganization.